shared-data/js/modules.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import moduleSpecs from '../robot-data/moduleSpecs.json'
+
+// use a name like 'magdeck' to get displayName for app
+export function getModuleDisplayName (name: string): ?string {
```

Comment:
Could we change these types for safety / convenience?

```suggestion
export function getModuleDisplayName (name: 'magdeck' | 'tempdeck'): string {
```

Annoyingly enough, flow imports JSON files as `any` instead of an object literal (https://github.com/facebook/flow/issues/4596) so we don't actually have any type safety in this function.

(FWIW TypeScript can [import well-formatted JSON as a typed object](https://blogs.msdn.microsoft.com/typescript/2018/05/31/announcing-typescript-2-9/#json-imports))

Reply:
For labware/pipette specs we have a flow def for the files (`LabwareDefinition` / `PipetteNameSpecs`), we oughta do that for module specs at some point